code/modules/mob/dead/observer/observer.dm

```diff
@@ -135,15 +135,19 @@
 	return ghost
 
 
-/mob/proc/set_away_time()
+/mob/proc/set_away_time(var/new_away)
 	return
 
-/mob/living/set_away_time()
+/mob/living/set_away_time(var/new_away = world.time)
+	if(!new_away)
+		return
 	away_time = world.time //Generic way to handle away time, currently unused.
 
 
-/mob/living/carbon/Xenomorph/set_away_time()
-	away_time = -XENO_AFK_TIMER //Xenos who force-ghost can be immediately taken by observers.
+/mob/living/carbon/Xenomorph/set_away_time(var/new_away = -XENO_AFK_TIMER)
```

Comment:
```suggestion
/mob/living/carbon/Xenomorph/set_away_time(new_away = -XENO_AFK_TIMER)
```
`-XENO_AFK_TIMER` will be -5 minutes, is this what you want?

Reply:
Yes because by default they are instantly takeover-able if they force ghost.
(was the original implementation.